js/news.json

```diff
@@ -1,5 +1,15 @@
 {
     "newsContent": [
+        {
+            "id": "54",
+            "newsId": "news54",
+            "newsTitle": "ONNX v1.8 Released",
+            "newsShortTitle": "ONNX v1.8 Released",
+            "newsDescription": "ONNX v1.8 is now available with exciting enhanced features! Windows conda package is also available in v1.8.0 Release (last supported version was v1.1.1). Training are enhanced by adding differentiable tags and removing GraphCall. Shape Inference and Checker become more robust and support models larger than 2GB. More operators are supported by Version Converter. Optimizer will move to another repo onnx/optimizer since next ONNX 1.9.0 Release. ",
```

Comment:
Adding some updates: 
ONNX v1.8 is now available with exciting enhanced features! Opset13 added with support for bfloat16 among other updates. Windows conda package added after v1.1.1. Added differentiable tags to enhance training scenario. Numerous updates to shape inference and checker tools. Updates to version converter to improve operator coverage.

Reply:
Looks better. Thank you.